Aptoide/aptoide-client-v8
# [ASV-1697] Seasonal Promotions

**What does this PR do?**

  This PR aims at changing the icon, the notification icon and title of the new promotion

**Database changed?**

 No

**Where should the reviewer start?**

- [ ] PromotionsFragment.java

**How should this be manually tested?**

  Rewrite promotions and open the app. You will see the dialog of the promotions and you if you press the "check it out" button or the icon on the top you will navigate to the promotions view where you should have the new image.

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1697](https://aptoide.atlassian.net/browse/ASV-1697)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
